__tests__/test_part1.py

```diff
@@ -0,0 +1,148 @@
+"""This file tests the following functions:
+
+    * get_maggy
+    * get_maggy_inv_var
+    * get_rest_mag
+    * get_volume
+    * get_binned_phi
+    
+"""
+
+# -----------------------
+# Package Imports
+# -----------------------
+import math 
+import numpy as np
+import pandas as pd
+import cv2
+# import pytest
+# import sys
+# sys.path.insert(1, '../lumfunc/')
+import lumfunc as lf
```

Comment:
This will use the version of `lumfunc` installed by pip on the system and therefore it might not have all the functions that you've integrated especially the `v2` functions which are not on pip but are in the repo.